components/dash-core-components/tests/integration/markdown/test_markdown.py

```diff
@@ -93,3 +93,56 @@ def test_mkdw002_dcclink(dash_dcc):
     dash_dcc.percy_snapshot("mkdw002 - markdowns display")
 
     assert dash_dcc.get_logs() == []
+
+
+def test_mkdw003_without_mathjax(dash_dcc):
+    app = Dash(__name__, eager_loading=False, assets_folder="../../assets")
```

Comment:
```suggestion
    app = Dash(__name__)
```
Best to leave these extras out unless you're explicitly testing them 😄